huvr_client/huvr_client.py

```diff
@@ -160,6 +160,32 @@ def project(self, data, id=None):
         else:
             return (response.status_code, response.text)
 
+    def delete_project(self, id):
+        """
+        DELETE /api/projects/<id>
+        """
+        url = "{}/api/projects/{}".format(self.base_url, id)
+        response = requests.delete(url)
```

Comment:
I think it needs to be like this:
self.client.delete(url)

a new call to requests will not have the session cookie

Reply:
fixed